desktop/package.json

```diff
@@ -13,21 +13,28 @@
 		"email": "wp-desktop@automattic.com"
 	},
 	"homepage": "https://desktop.wordpress.com/",
-	"main": "build/desktop.js",
+	"main": "./dist/index.js",
 	"keywords": [
 		"desktop",
 		"wordpress",
 		"wordpress.com"
 	],
 	"scripts": {
-		"build": "npx electron-builder ${ELECTRON_BUILDER_ARGS} build --publish never",
-		"ci:build-mac": "node bin/build-mac-ci",
+		"dev": "cross-env NODE_ENV=development DEBUG=1 WP_DESKTOP_DEBUG=1 electron app/index.js",
+		"dev:localhost": "cross-env WP_DESKTOP_DEBUG_LOCALHOST=1 yarn run dev",
+		"ci:build-mac": "yarn run build:main && node bin/build-mac-ci",
+		"build": "yarn run build:main && ELECTRON_BUILDER_ARGS=${ELECTRON_BUILDER_ARGS} yarn run build:app",
```

Comment:
Isn't `ELECTRON_BUILDER_ARGS=${ELECTRON_BUILDER_ARGS}` redundant? Looks like assigning a variable to itself.

Reply:
Yeah it seems odd, but if you exclude the variable assignment, e.g. `yarn run build:main && ${ELECTRON_BUILDER_ARGS} yarn run build:app`, then the build job will fail with:

```
$ yarn run build:main && ${ELECTRON_BUILDER_ARGS} yarn run build:app
$ webpack --display errors-only
/bin/sh: -c.mac.target=dir: command not found
```

At this point the script needs to pass the argument to the nested `build` script within a parent variable, and not directly. This is because the nested build script is defined as:

```
"build": "yarn run build:main && ${ELECTRON_BUILDER_ARGS} yarn run build:app",
```